packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -448,25 +442,28 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     // Instead of setting filters one by one for each link,
     // combine them so they could be set in a single call per target panel
     for (let i = 0; i < links.length; i += 1) {
-      const { start, end, comparisonOperator } = links[i];
+      const { start, end, operator } = links[i];
       if (start.panelId === panelId && end != null) {
         const { panelId: endPanelId, columnName, columnType } = end;
         // Map of column name to column type and filter value
         const filterMap = panelFilterMap.has(endPanelId)
           ? panelFilterMap.get(endPanelId)
           : new Map();
         const { value } = dataMap[start.columnName];
-        const operator =
-          columnType != null && TableUtils.isStringType(columnType)
-            ? getSymbolForTextFilter(comparisonOperator ?? Type.eq)
-            : getSymbolForNumberOrDateFilter(comparisonOperator ?? Type.eq);
-
-        let text = `${operator}${value}`;
-        if (comparisonOperator === 'startsWith') {
-          text = `${value}${operator}`;
+        let symbol = '';
+        if (operator !== undefined && operator !== 'eq') {
+          if (operator === 'startsWith' || operator === 'endsWith') {
+            symbol = '*';
+          } else {
+            symbol = TableUtils.getFilterOperatorString(operator);
+          }
+        }
+        let text = `${symbol}${value}`;
+        if (operator === 'startsWith') {
+          text = `${value}${symbol}`;
```

Comment:
Don't do all this operator stuff here. Instead, just pass the `operator` through in the `filterMap` (along with `columnType`, `text`, `value`) and then in `IrisGridPanel` or `IrisGrid` build the text there (as you'll have the array of filters as well).